RdlDesign/ReportDesigner.csproj

```diff
@@ -118,6 +118,10 @@
     <WarningLevel>4</WarningLevel>
   </PropertyGroup>
   <ItemGroup>
+    <Reference Include="Devart.Data.PostgreSql, Version=7.4.506.0, Culture=neutral, PublicKeyToken=09af7300eec23701, processorArchitecture=MSIL">
+      <SpecificVersion>False</SpecificVersion>
+      <HintPath>..\..\..\..\..\..\..\..\Program Files (x86)\Devart\dotConnect\PostgreSQL\Devart.Data.PostgreSql.dll</HintPath>
```

Comment:
This is going to cause a problem. It expects the library to be installed on all developers computers. 

Can you update the project to use the http://www.nuget.org/packages/dotConnect.Express.for.PostgreSQL/ nuget package.